### ISSUE TITLE
private ap ipv6 prefix

### ISSUE THREAD

**Issue by mkg20001** [ffgraz/gluon#48]
gets announced via hna, new range for private ap ipv6 prefixes, something like /48

**Comment by mkg20001** [comment]
not sure if we can actually do this since we will just get /44 from edis

**Comment by nicoonoclaste** [comment]
Not sure what you mean here.  Do you want a distinct IPv6 prefix for each node?
Presumably to announce for a LAN with non-OLSR devices ?

**Comment by mkg20001** [comment]
exactly. but i'm not sure if we can do that, considering we will get at most a /44 from edis. since for ra we'll need a /64

**Comment by nicoonoclaste** [comment]
> exactly. but i'm not sure if we can do that, considering we will get at most a /44 from edis.

in principle, we could get allocations from RIPE, but i don't see that being necessary: a /44 still contains 2²⁰ = 1 048 576 separate /64s.

if we ever get to 10k nodes I'll break out the champagne, let alone a million
... but even then, we can give out prefixes as requested, rather than give each device a /60 or something.

> since for ra we'll need a /64

Theoretically not necessary, but yes I'm probably the only person ornery-enough to bother with prefixes longer than /64 (and cackle when bad software gets reaaaally sad about it)
